### DISCUSSION TITLE
about save twice problem

### DISCUSSION THREAD

**Discussion by Tback1** [coddingtonbear/obsidian-web#176] [Bug Report]
### Describe the bug

我发现这个问题一个可能的原因，但不是每次都能复现
挺大概率是当保存的网页title有一些字符类似 `|` `-`等的没有被正确处理，在本地生成的文件名称为空白，即一个 `.md` 的文件。而再保存一次，这个文件会被改名成正确的名字，而第二次保存的文件就被直接生成了一个副本

### To Reproduce

2. Click on '...'
3. Scroll down to '...'

### Operating System

win10 LTSC

### Browser

Chrome 

### Browser Version

120

**Comment by coddingtonbear** [comment]
Please don't open multiple discussions for the same issue.  I imagine that you are hoping that doing so might get my attention more quickly, but unfortunately the reason I haven't given this issue attention isn't due to my needing a reminder: I just haven't experienced any issues of this nature myself, and I haven't found the time to look into this yet.

**Reply by Tback1** [reply]
I know, and I've put related possible issues in the same issue before, but your suggestion is that it's better to separate,
And this time I can only say that there are actually three issues(or phenomena) involved(maybe save twice, duplication error, error save file name.)
So I opened new one to Express my concerns
